ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb

```diff
@@ -0,0 +1,1073 @@
+{
```

Comment:
<p>Is there a way to clean up whatever <code>log_metrics</code>  creates?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Reply:
Same as above.